docs/modules/deployment/pages/time-series-storage/timeseries/time-series-storage.adoc

```diff
@@ -65,9 +70,11 @@ opennms-newts
 [source, console]
 sudo systemctl restart opennms
 
-== Write to Time Series Integration Plugin in parallel
+[[ga-dual-write-integration]]
+=== Write to time series integration plugin in parallel
 
-You can enable writing to any time series integration plugin in parallel without having integration as the default time series strategy.
+You can enable writing to any time series integration plugin in parallel to RRD or Newts by enabling the `opennms-timeseries-api` feature and not setting the strategy to `integration`.
```

Comment:
In one other spot, we suggest setting it to `rrd`. We might want to be consistent. https://github.com/OpenNMS/opennms/pull/5788/files#diff-d7bbf1320c17520776181e8ec0528061d849967d21c1a02c826bb90a92203910R54

Reply:
see comment at linked line.